common/xbps-src/shutils/lint_commits.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+lint_commits() {
+	local base="$1"
+	local status=0
+	local cmt
+	if [ -n "$base" ]; then
+		base=$("$XBPS_GIT_CMD" rev-parse --verify "$base") ||
+			return
+	elif base=$("$XBPS_GIT_CMD" rev-parse --verify FETCH_HEAD 2>/dev/null); then
+		:
+	elif base=$("$XBPS_GIT_CMD" rev-parse --verify ORIG_HEAD 2>/dev/null); then
+		:
+	else
+		echo base commit not found >&2
```

Comment:
Use `msg_error`? (is it available here?)

Reply:
I haven't checked. However, as of it's now, it can't be run in Ubuntu since `xbps-src` needs to check for `xbps-*`.
I'm not sure if it's worth to be put inside `xbps-src`.